tests/test_routing.py

```diff
@@ -130,3 +130,46 @@ async def test_router_copy():
     await other_router.dispatch(event)
     assert deep_callback.called
     assert shallow_callback.called
+
+
+def test_fetch_callbacks():
+    router = routing.Router()
+    event = sansio.Event(
+        {"action": "opened", "count": 42, "status": "completed"},
+        event="yeah",
+        delivery_id="1234",
+    )
+    no_callbacks = router.fetch(event)
+    assert not no_callbacks  # No callbacks registered.
```

Comment:
The double negative throws me every time. 😄 

```suggestion
    callbacks = router.fetch(event)
    assert not callbacks  # No callbacks registered.
```